ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIComponent/SettingTimeView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  SettingTimeView.swift
+//
+//
+//  Created by SONG on 5/30/24.
+//
+
+import UIKit
+
+import ToDoGardenUIConstant
+import ToDoGardenUIResource
+
+public class SettingTimeView: UIView {
+  private var configuration: Configuration
+  private let timepicker: ToDoGardenTimePicker
+  
+  public init(with button: UIButton, for configuration: Configuration) {
+    self.configuration = configuration
+    self.timepicker = ToDoGardenTimePicker(configuration: self.configuration.dataStore.timePicker)
+    super.init(frame: CGRect.zero)
+    self.build(with: button)
+  }
+  
+  @available(*, unavailable)
+  public required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override public var intrinsicContentSize: CGSize {
+    let screenWidth = UIScreen.main.bounds.width
+    let screenHeight = UIScreen.main.bounds.height
```

Comment:
p2;

<img width="369" alt="Screenshot 2024-06-03 at 4 12 26 PM" src="https://github.com/ToDo-Garden/ToDo-Garden/assets/63908856/d9ab4cbb-49fc-4e51-b7ee-743b90a90ab6">

UIScreen.main 타입 프로퍼티는 곧 deprecated가 되니 아래에서 가이드하는 예시처럼 사용하는건 어떨까요?

```suggestion
    let windowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene
    let screenWidth = windowScene?.screen.bounds.width ?? 1.0
    let screenHeight = windowScene?.screen.bounds.height ?? 1.0
```